### ISSUE TITLE
Pkg.test demo "lines_scatter.jl" has a deprecation warning

### ISSUE THREAD

**Issue by CrashBurnRepeat** [JuliaGL/GLVisualize.jl#211]
When loading `lines_scatter.jl` in the demos, the following deprecation warning is thrown.

```
WARNING: map!(f::F, A::AbstractArray) where F is deprecated, use map!(f, A, A) instead.
Stacktrace:
 [1] depwarn(::String, ::Symbol) at ./deprecated.jl:70
 [2] map!(::GeometryTypes.MeshMulFunctor{Float64}, ::Array{GeometryTypes.Point{3,Float32},1}) at ./deprecated.jl:57
 [3] *(::StaticArrays.SArray{Tuple{4,4},Float64,2,16}, ::GeometryTypes.HomogenousMesh{GeometryTypes.Point{3,Float32},GeometryTypes.Face{3,GeometryTypes.OffsetInteger{-1,UInt32}},GeometryTypes.Normal{3,Float32},Void,Void,Void,Void}) at /Users/crashburnrepeat/.julia/v0.6/GeometryTypes/src/meshes.jl:223
 [4] include_from_node1(::String) at ./loading.jl:576
 [5] include(::String) at ./sysimg.jl:14
 [6] _test_include(::String, ::ExampleRunner.RunnerConfig) at /Users/crashburnrepeat/.julia/v0.6/GLVisualize/src/../examples/ExampleRunner.jl:323
 [7] inner_test(::String, ::ExampleRunner.RunnerConfig, ::GLWindow.Screen, ::Base.RefValue{Bool}, ::Base.RefValue{Int64}, ::ExampleRunner.#increase#23{Base.RefValue{Int64}}) at /Users/crashburnrepeat/.julia/v0.6/GLVisualize/src/../examples/ExampleRunner.jl:376
 [8] make_tests(::ExampleRunner.RunnerConfig) at /Users/crashburnrepeat/.julia/v0.6/GLVisualize/src/../examples/ExampleRunner.jl:456
 [9] run(::ExampleRunner.RunnerConfig) at /Users/crashburnrepeat/.julia/v0.6/GLVisualize/src/../examples/ExampleRunner.jl:482
 [10] include_from_node1(::String) at ./loading.jl:576
 [11] include(::String) at ./sysimg.jl:14
 [12] include_from_node1(::String) at ./loading.jl:576
 [13] include(::String) at ./sysimg.jl:14
 [14] process_options(::Base.JLOptions) at ./client.jl:305
 [15] _start() at ./client.jl:371
while loading /Users/crashburnrepeat/.julia/v0.6/GLVisualize/examples/plots/lines_scatter.jl, in expression starting on line 19
```

**Comment by CrashBurnRepeat** [comment]
I should have specified this occurs when running `3dplots.jl` through the `Pkg.test` interface.

**Comment by CrashBurnRepeat** [comment]
Addressed in [issue #118 of GeometryTypes](https://github.com/JuliaGeometry/GeometryTypes.jl/issues/118), which I've created a pull request for.
